apps/src/applab/firebaseCounters.js

```diff
@@ -35,7 +37,7 @@ export function updateTableCounters(tableName, rowCountChange, updateNextId) {
       }
       tableData.lastId = (tableData.lastId || 0) + 1;
     }
-    if (tableData.rowCount + rowCountChange > TABLE_ROW_COUNT_LIMIT)  {
+    if (tableData.rowCount + rowCountChange > tableRowCountLimit)  {
```

Comment:
`updateTableCounters` uses a constant - does it also need to be wrapped in `loadConstants().then(() => {...})`?